app/routes/collection/add.js

```diff
@@ -2,13 +2,29 @@ import Ember from 'ember';
 
 
 export default Ember.Route.extend({
+
     panelActions: Ember.inject.service('panelActions'),
+    caxe: Ember.inject.service(),
+
     model() {
         const collectionType = this.modelFor('collection').get('collectionType');
-        return Ember.RSVP.hash({
-            workflow: this.store.findRecord('workflow', collectionType),
-            collection: this.modelFor('collection'),
-        });
+        let submissionFormId = {
+            'meeting': 1
+        }[collectionType];
+        const caxe = this.store.createRecord('case');
+        return this.store.findRecord('workflow', submissionFormId)
```

Comment:
Get the workflow, and then create a new case for that workflow. Will need some more logic here in the future to account for loading previously started cases.